pkg/stub/handler.go

```diff
@@ -24,27 +25,26 @@ const (
 	clusterConfigResource = "cluster-config-v1"
 )
 
-func NewHandler() *Handler {
+func NewHandler(namespace string, manifestFactory *manifests.Factory) *Handler {
 	return &Handler{
-		manifestFactory: manifests.NewFactory(),
+		namespace:       namespace,
+		manifestFactory: manifestFactory,
 	}
 }
 
 type Handler struct {
+	namespace       string
 	manifestFactory *manifests.Factory
 }
 
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
+	// TODO: This should be adding an item to a rate limited work queue, but for
+	// now correctness is more important than performance.
 	switch o := event.Object.(type) {
 	case *ingressv1alpha1.ClusterIngress:
-		if event.Deleted {
-			logrus.Infof("Deleting ClusterIngress object: %s", o.Name)
-			return h.deleteIngress(o)
-		} else {
-			return h.syncIngressUpdate(o)
-		}
+		logrus.Infof("reconciling for update to clusteringress %q", o.Name)
 	}
-	return nil
+	return h.reconcile()
```

Comment:
perform h.reconcole() only when we are dealing with ingressv1alpha1.ClusterIngress? This seems to call this reconciler for non cluster ingress objects.

Reply:
It'll trigger for any object we're actively watching in our entry point, which for now is just ClusterIngress, but in the future should be other types of resources that we might want to trigger a reconciliation.

Reply:
Why do we log conditionally and reconcile unconditionally?

Reply:
No reason, logging can be tweaked